src/main/java/org/cactoos/list/ListNoNulls.java

```diff
@@ -28,6 +28,8 @@
 import java.util.List;
 import java.util.ListIterator;
 import org.cactoos.collection.CollectionNoNulls;
+import org.cactoos.text.FormattedText;
+import org.cactoos.text.UncheckedText;
 
```

Comment:
@proshin-roman Shouldn't `ListIterator` instances returned in `listIterator()` and `listIterator(final int index)` be null-free too?

Reply:
@paulodamaso I'm sure it's not related to the ticket's theme :) You can submit a new bug if you want.

Reply:
@proshin-roman You're right, I'll submit a bug, thanks.